modules/installation-osp-deploying-all-bare-metal-preexisting.adoc

```diff
@@ -0,0 +1,70 @@
+[id="installation-osp-deploying-all-bare-metal-preexisting_{context}"]
+= Configuring a cluster to use bare-metal control plane and compute machines
+
+If you want your cluster to use bare-metal control plane and compute machines, modify the
+`install-config.yaml` file. In this configuration, control plane and compute machines are
+deployed on user-provisioned infrastructure.
+
+[NOTE]
+====
+Bare-metal compute machines are not supported on clusters that use Kuryr.
+====
+
+.Prerequisites
+
+* The {rh-openstack} link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/bare_metal_provisioning/index[Bare Metal service (Ironic)] is enabled and accessible by using the {rh-openstack} Compute API.
+
+* Bare metal is available as link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/bare_metal_provisioning/sect-configure#creating_the_bare_metal_flavor[an {rh-openstack} flavor].
+
+* {rh-openstack} is running on network infrastructure that you provisioned.
+
+* The {rh-openstack} network supports both VM and bare metal server attachment.
```

Comment:
This case uses bare-metal machines only. This way, we get rid of the requirement of a network that supports both types (which is hard to fulfil). This line could be:
```
The {rh-openstack} network supports bare metal server attachment.
```

Reply:
@pierreprinetti Whoops--missed editing that when I repurposed the previous module. Fixed.